p5/io/local_data.py

```diff
@@ -52,27 +55,38 @@ def get_column(self,name):
 
 	def get_row(self,index):
 		"""
-		Returns an entire row when given the row index.
+		:param index: Index of the required row.
+		:type index: int
 
-		:param index: Name of the row
-		:type index: string
+		:returns: An entire row when given the row index.
+		:rtype: list
 		"""
-		for fragment in self.data:
-			if fragment[0] == index:
-				return fragment
+		return self.data[index]
 	def get_array(self):
 		"""
-		Returns the entire csv data as an multidimensional array.
+		:returns: the entire csv data as an multidimensional array.
+		:rtype: list
 		"""
 		return self.data
 
 
-def load_table(path):
+def load_table(path,mode):
```

Comment:
p5.js has other optional arguments too like header, error, and success callbacks. You can use *args to take multiple optional variables, would-be future compatible. So something like 
```
def load_table(path, *args):
    table = Table(path, *args)
```